internal/access/signup.go

```diff
@@ -13,38 +15,39 @@ import (
 
 // Signup creates a user identity using the supplied name and password and
 // grants the identity "admin" access to Infra.
-func Signup(c *gin.Context, name, password, orgName string) (*models.Identity, *models.Organization, error) {
+func Signup(c *gin.Context, keyExpiresAt time.Time, name, password string, org *models.Organization) (*models.Identity, string, error) {
```

Comment:
We'll probably want to create a struct to accept all these arguments pretty soon, but not a blocker